Failure to recognise optional 'let _'

<!-- Note: If instead of reporting a bug you're looking for help using Gryphon, you can always send a message [on Twitter](https://twitter.com/gryphonblog). -->

**Describe the bug**
Using templates, Gryphon should translate the following successfully:

```swift
private func gryphonTemplates() {
    let _uuid = UUID()
    
    let _ = _uuid.uuidString
    let _ = "_uuid.toString()"
}
```

so that something like this:

```swift
id = UUID().uuidString
```

turns into this:

```kotlin
id = UUID().toString()
```

However, the following error is thrown, failing the translation:
```bash
🚨 File.swift:19:5: error: failed to turn Swift AST into Gryphon AST: Type not recognized.
    let _ = _uuid.uuidString
    ^~~~~~~~~~~~~~
``` 

_Removing the `let` makes the translation succeed. This should work as it compiles fine in Swift._

**How to reproduce it**
1. Create `gryphonTemplates` as seen above.
2. Run gryphon translation.
3. Build failed.

**Your environment (please complete the following information):**
 - OS: macOS 10.15.5
 - Gryphon version 0.9
 - Xcode 11.5 (Swift 5.2)

--- a/Sources/GryphonLib/SwiftTranslator.swift
+++ b/Sources/GryphonLib/SwiftTranslator.swift
@@ -315,8 +315,18 @@ public class SwiftTranslator {
 		case "Defer Statement":
 			result = [try translateDeferStatement(subtree)]
 		case "Pattern Binding Declaration":
-			try processPatternBindingDeclaration(subtree)
-			result = []
+			// If it's a `let _ = <expression>`
+			if subtree.subtrees.count == 2,
+				subtree.subtrees[0].name == "Pattern Any"
+			{
+				result = try [ExpressionStatement(
+					range: getRangeRecursively(ofNode: subtree),
+					expression: translateExpression(subtree.subtrees[1]))]
+			}
+			else {
+				try processPatternBindingDeclaration(subtree)
+				result = []
+			}
 		case "Return Statement":
 			result = [try translateReturnStatement(subtree)]
 		case "Break Statement":

--- a/Test cases/misc.kt
+++ b/Test cases/misc.kt
@@ -92,4 +92,6 @@ fun main(args: Array<String>) {
 	val arrayIndex: Int? = array.indexOf(1)
 	val bla: Int = 1
 	var foo: (() -> Unit)? = null
+
+	"abc"
 }

--- a/Test cases/misc.swift
+++ b/Test cases/misc.swift
@@ -119,3 +119,6 @@ class E: CustomStringConvertible {
 class F: CustomStringConvertible {
 	var description: String = "abc"
 }
+
+// Anonymous pattern binding
+let _ = "abc"

